.travis.yml

```diff
@@ -31,5 +30,8 @@ before_script:
   - cd ..
 
 script:
-  - cmake -DOPENSSL_ROOT_DIR=${PREFIX} -DOPENSSL_LIBRARIES=${PREFIX}/lib && make
-  - prove -v test/{00-engine.t,01-digest.t,02-mac.t,03-encrypt.t}
```

Comment:
you have removed `prove` tests, where are they added in your code ?

Reply:
`make test` runs the `prove` tests.

`make test` runs `perl run_tests` https://github.com/gost-engine/engine/blob/master/CMakeLists.txt#L131
`perl run_tests` runs `prove` https://github.com/gost-engine/engine/blob/master/test/run_tests (`prove` is `TAP::Harness`)

Reply:
`04` was failing for me (it was working fine when running locally)
does it run well in travis now ?

Reply:
Also, you can see that those tests are run by looking at the Travis CI output: https://travis-ci.org/gost-engine/engine/builds/448126285#L595

Reply:
you mean `4/4` ?

Reply:
Right. According to https://github.com/gost-engine/engine/blob/master/CMakeLists.txt#L131 `perl run_tests` is known to cmake with the name `engine`. You can see in Travis that the test named `engine` passed.